conf/config-default.yaml

```diff
@@ -382,6 +382,7 @@ plugins:                          # plugin list (sorted by priority)
   - syslog                         # priority: 401
   - udp-logger                     # priority: 400
   - file-logger                    # priority: 399
+  #- clickhouse-logger             # priority: 398
```

Comment:
Looks like it's safe to turn on the logger by default?

Reply:
It is safe. The principal of plugins is enabled by default ?

Reply:
This plugin doesn't require any setup and doesn't affect other components, so I think it's safe enough.